### ISSUE TITLE
Getting an error when using inputMask with MSIE

### ISSUE THREAD

**Issue by mars6277** [RobinHerbots/Inputmask#346]
Recently upgraded from an earlier version of inputMask.js.  Found that our widgets were getting errors in MSIE 10.  had a look at the files and found in inputmask.js line 84 it read:
 "msie10 = /_@cc_on!@_/false".

Not sure that this was intentional.

To fix our situation I changed it in our local copy to read(as in the previous version we had, 2.2.47):
 "msie10 = navigator.userAgent.match(new RegExp("msie 10", "i")) !== null,"

Resolved our issue, Though someone might like to know about it.

**Comment by RobinHerbots** [comment]
@mars6277 , 

Can you be a bit more specific about the error you were getting?
I do not get errors on IE10 with the above line.

**Comment by mars6277** [comment]
Thank you for getting back to me.  Sorry I wasn’t more specific.

Running our web app with the “msie10 = /_@cc_on!@_/false” (line 84 of jquery.inputmask.js version 2.3.50) we are getting the error SCRIPT5017: Syntax error in regular expression.  We didn’t realize anything was happening till we noticed a widget that was not appearing correctly in Internet Explorer 10.  The widget itself wasn’t using inputmask but the failure was occurring before the widget was getting rendered.

I went through our repos to determine when it was introduced and found that it happened when we upgraded from inputmask 2.2.47 to 2.3.50.  From there I tracked it to line 84.  When I reverted the line to read as it did in 2.2.47 “msie10 = “navigator.userAgent.match(new RegExp("msie 10", "i")) !== null” the SCRIPT5017 isn’t produced and our widget renders.

I suspect that “msie10 = false” makes it not function at all in IE 10.

I’m sorry I didn’t do much more digging than that.

If I may say our group is pleased with the  inputmask, thank you for providing it!  If you would like me to look at it more let me know.
